pdata/internal/metrics_test.go

```diff
@@ -691,15 +691,20 @@ func TestMetricsClone(t *testing.T) {
 func TestMetricsDataPointFlags(t *testing.T) {
 	gauge := generateTestGauge()
 
-	gauge.DataPoints().At(0).SetFlags(NewMetricDataPointFlags())
+	gauge.DataPoints().At(0).SetFlags(NewEmptyMetricDataPointFlags())
```

Comment:
No need to set this correct?

Reply:
Technically it works with the original function, but Goland got all upset bc it was deprecated.  If you want to keep it so that the deprecation/transition process has less modifications thats find with me.